lib/cocoapods-search/command/search.rb

```diff
@@ -22,6 +22,7 @@ def self.options
         options += Platform.all.map do |platform|
           ["--#{platform.name.to_s}",     "Restricts the search to Pods supported on #{Platform.string_name(platform.to_sym)}"]
         end
+        options << ['--no-pager',    'Do not pipe search results into a pager']
```

Comment:
I think this is preferred by git because of the existence of environment variable `$PAGER`. For positive options you might be right, but people really wouldn't use positive option because it's the default.

Let's stay with this if you don't mind much?